go/libkb/secret_store_linux.go

```diff
@@ -31,5 +31,5 @@ func NewSecretStoreAll(mctx MetaContext) SecretStoreAll {
 		// Use system keychain but fall back to file store if not available.
 		return SecretStoreFallbackBehaviorOnError
 	}
-	return NewSecretStoreUpgradeable(ssecretservice, sfile, shouldUpgradeOpportunistically, shouldStoreInFallback)
+	return NewSecretStoreUpgradeable(ssecretservice, sfile, "system keyring", "file-based secret store (see https://keybase.io/docs/crypto/local-key-security)", shouldUpgradeOpportunistically, shouldStoreInFallback)
```

Comment:
do we need the keybase.io url here? does user see this somewhere?

Reply:
I'm not sure, @heronhaye suggested this

Reply:
user sees it in warning when during fallback on signup

Reply:
wouldn't this also cause long and less readable long lines?

Reply:
Maybe. Do you have a suggestion on how to fix it? We could pass in a third "info string" to the struct that is only printed in user-facing logs. Imo it's fine as is, up to you

Reply:
let's go with this and figure it out later.

I wonder if typical user even sees it, this does not plumb up to the gui, right?

Or is it more for the cli users?

Reply:
For CLI users. Good point about GUI, idk.